testbed/tests/metric_test.go

```diff
@@ -46,6 +37,24 @@ func TestMetric10kDPS(t *testing.T) {
 				ExpectedMaxRAM: 59,
 			},
 		},
+		{
+			"Carbon",
+			NewCarbonDataSender(testbed.GetAvailablePort(t)),
+			NewCarbonDataReceiver(testbed.GetAvailablePort(t)),
+			testbed.ResourceSpec{
+				ExpectedMaxCPU: 115,
```

Comment:
Does carbon use a lot more CPU than OC or SFx?

Reply:
In the current e2e tests yes: the profile is dominated by calls to "poll" and writing to the TCP connection. Clearly it could benefit from a processor batching the metrics so the exporter could have fewer writes on the connection.